SCF-Session-8-Assignment/src/question/Cage.java

```diff
@@ -0,0 +1,28 @@
+package question;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class is for the cages in the zones
+ * @author Jugal
+ * Dated 07/24/2019
+ */
+public class Cage {
+	
+	 int cageCapacity;

```

Comment:
Declare the variables using access modifiers.

Reply:
Given the package access for the variable.